libsovtoken/src/logic/parsers/parse_get_utxo_response.rs

```diff
@@ -130,14 +132,48 @@ pub fn get_utxo_state_proof_extractor(reply_from_node: *const c_char, parsed_sp:
         None => return ErrorCode::CommonInvalidStructure
     };
 
+    let from: Option<u64> = match result.get(FROM) {
```

Comment:
I think it worth to simplify all this code but it seems like out of scope for this PR